packages/i18n/fr.json

```diff
@@ -317,13 +320,63 @@
       "explanationRemovingGroups": "Il est également possible de supprimer un ou plusieurs groupes pour cette personne en sélectionnant les groupes dans la colonne de droite.",
       "noUserFound": "Aucune personne trouvée",
       "noGroupFound": "Aucun groupe trouvé",
-      "result": "Affichage de {elements} groupes sur {total}."
+      "result": "Affichage de maximum {elements} groupes."
     },
     "confirmUpdateEmail": {
       "title": "Mettre à jour son email",
       "message": "Cette opération a été effectuée avec succès. Vous pouvez maintenant vous connecter avec votre nouvel email.",
       "errorMessage": "Malheureusement, l'opération n'a pas pu être menée à bien. Si le problème persiste, veuillez contacter l'administrateur du site.",
       "signIn": "Se connecter"
+    },
+    "adminGroups": {
+      "title": "@:pages.admin.groups",
+      "addGroup": "Ajouter un groupe",
+      "emptyResult": "Aucun résultat trouvé",
+      "search": "Commencez la saisie pour rechercher…",
+      "result": "Affichage de {elements} groupes sur {total}.",
```

Comment:
If there is only one `elements`, could you use pluralization https://vue-i18n.intlify.dev/guide/essentials/pluralization.html ? (same for en translation, `adminGroups.result`, `recordUser.result`, `recordGroup.result` keys)

Reply:
I will do it for `adminGroups` and `adminUsers` only because it is not relevant for the others